e2e/rbd_helper.go

```diff
@@ -232,6 +234,162 @@ func kmsIsVault(kms string) bool {
 	return kms == "vault"
 }
 
+// nolint:gocyclo // reduce complexity
```

Comment:
Please use helper function from the start. Our e2e code should have the same quality as the other components.